docs/manual-EN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md

```diff
@@ -0,0 +1,44 @@
+# String Comparison Functions and Operators
+
+| Name     | Description          |
+|:-----    | :------------------: |
+| CONTAINS | Perform case-sensitive inclusion searching in strings |
+
+* CONTAINS
+
+The `CONTAINS` operator is used to perform case-sensitive matching regardless of location within a string. All non-string type are forced to convert to strings. The `CONTAINS` operator requires string type in both left and right side.
```

Comment:
R U sure about the "All non-string type are forced to convert to strings"
I don't think so.